packages/cli/src/commands/create/astro-config.ts

```diff
@@ -171,6 +171,8 @@ function updateObject(properties: any, object: t.ObjectExpression | undefined):
       }
     }
   }
+
+  return object;
```

Comment:
Oh interesting, this just made visible that the return value is not used. We should probably have line `154` just do a `return`.

Reply:
Yep I was wondering the same cause the return value is not used but I wasn't sure and I didn't wanna change it too much.

Reply:
Fixed it.

Reply:
Yay!! :star_struck: